front-end/src/App.js

```diff
@@ -0,0 +1,16 @@
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom'; 
+import HomePage from './pages/HomePage/HomePage.jsx';
```

Comment:
If you notice, this import has ..Homepage/Homepage, it has duplicate "Homepage" which leading the code has a little confusion when imported. You can add an index.js file contain "export * from './Homepage'; ". Then import it again at other files.